build/package/generate-msi.sh

```diff
@@ -53,4 +53,7 @@ go run ./tools/sign -file "dist/${PACKAGE_NAME}"
 
 if [[ "${TOOL_NAME:?}" == atlascli ]]; then
   go run ./tools/chocolateypkg/main.go --version "${VERSION}" --file "dist/${PACKAGE_NAME}" --url https://fastdl.mongodb.org/mongocli/"${PACKAGE_NAME}"
+  if [[ "${RELEASING:?}" == true ]]; then
+      go run ./tools/chocolateyupdate/main.go --version "${VERSION}"
```

Comment:
Let's pass the path instead of having logic on the go code

Reply:
This path? I'm asking, because I have also defined it like that in the previous script. `const path = "build/package/chocolatey"`